components/FeatureHighlight/src/FeatureHighlightAccessibilityMutator/MDCFeatureHighlightAccessibilityMutator.m

```diff
@@ -0,0 +1,66 @@
+/*
+ Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "MDCFeatureHighlightAccessibilityMutator.h"
+#import "MaterialFeatureHighlight.h"
```

Comment:
Nit: Please sort imports alphabetically.

Reply:
I thought the guide line is to put interface of the class on top?

Reply:
Yes, I believe either:

*  "<MDF..." comes before "Material..." (capital ordering first)
* "<MDF..." comes last since it is a separate project

I think randallli@ knows best.